operator/Makefile

```diff
@@ -67,7 +67,8 @@ docker-build: test ## Build docker image with the manager.
 	docker build -t ${IMG} .
 
 docker-push: ## Push docker image with the manager.
-	docker push ${IMG}
+	docker tag ${IMG} localhost:37129/${IMG}
```

Comment:
this is very specific to your env but we are going to be redoing all this next sprint so we can leave it for now.

Reply:
ok I will open issue for this